plugins/clipboard-manager/guest-js/index.ts

```diff
@@ -53,4 +55,46 @@ async function readText(): Promise<string> {
   return kind.plainText.text;
 }
 
-export { writeText, readText };
+/**
+ * Writes HTML or fallbacks to write provided plain text to the clipboard.
+ * @example
+ * ```typescript
+ * import { writeHtml, readHtml } from '@tauri-apps/plugin-clipboard-manager';
+ * await writeHtml('<h1>Tauri is awesome!</h1>', 'plaintext');
+ * await writeHtml('<h1>Tauri is awesome!</h1>', '<h1>Tauri is awesome</h1>'); // Will write "<h1>Tauri is awesome</h1>" as plain text
+ * assert(await readHtml(), '<h1>Tauri is awesome!</h1>');
```

Comment:
we should use readText() here 
```suggestion
 * assert(await readText(), '<h1>Tauri is awesome!</h1>');
```

Reply:
Resolved